research/object_detection/box_coders/faster_rcnn_box_coder.py

```diff
@@ -107,8 +107,10 @@ def _decode(self, rel_codes, anchors):
       tx /= self._scale_factors[1]
       th /= self._scale_factors[2]
       tw /= self._scale_factors[3]
-    w = tf.exp(tf.minimum(tw, 80)) * wa
-    h = tf.exp(tf.minimum(th, 80)) * ha
+    # Avoid INF in exp and w,h below. We set 80 as a upper_bound since tf.exp(89) yields INF.
+    UPPER_BOUND = 80
```

Comment:
Could you please make the name more specific (e.g. UPPER_BOUND_OF_XXX) and move it to the top of this file(near line 36) ?

Reply:
Sure. I've done it.